api/src/neo4j/queries/map.js

```diff
@@ -1,8 +1,40 @@
 import querySingleResult from 'neo4j/queryHandlers/single';
 import queryListResult from 'neo4j/queryHandlers/list';
 import parseParams from 'neo4j/shared/helper';
+const fs = require('fs');
+const readline = require('readline');
```

Comment:
These could be changed to `import` for consistency.

```suggestion
import fs from 'fs';
import readline from 'readline';
```